packages/embark-ui/src/components/ContractTransactions.js

```diff
@@ -121,9 +121,14 @@ class ContractTransactions extends React.Component {
               <tbody>
                 {
                   this.dataToDisplay().map((log, index) => {
+                    let debugButton;
+                    if(log.transactionHash !== undefined) {
+                      debugButton = <DebugButton forceDebuggable transaction={{hash: log.transactionHash}}/>;
+                    }
+
                     return (
                       <tr key={'log-' + index}>
-                        <td><DebugButton forceDebuggable transaction={{hash: log.transactionHash}}/></td>
+                        <td>{debugButton}</td>
```

Comment:
Cooler way to do it in react:
```suggestion
                        {log.transactionHash && <td><DebugButton forceDebuggable transaction={{hash: log.transactionHash}}/></td>}
```

Reply:
TIL!

Might be a moot point, as @michaelsbradleyjr's PR touches this stuff too and he's shuffled it around some.

Reply:
Done now in #1392.